x-pack/legacy/plugins/rollup/public/services/api.js

```diff
@@ -3,15 +3,12 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
-import chrome from 'ui/chrome';
 let httpClient;
 export const setHttpClient = client => {
   httpClient = client;
 };
-const apiPrefix = chrome.addBasePath('/api/rollup');
 
 export async function getRollupIndices() {
-  const response = await httpClient.get(`${apiPrefix}/indices`);
+  const response = await httpClient.get(`api/rollup/indices`);
```

Comment:
Using `chrome.addBasePath()` adds base path one more time. Because `core.http` does it, too. 

So, `ui/chrome` is removed.